### PR TITLE
Remove duplicate entries in cardpostfixnametable.txt

### PR DESCRIPTION
**Pull request by Daraen1**
Remove duplicate entries in cardpostfixnametable.txt

issue #27 

I was bored so I decided to do it. Tell me if something wrong.
Thanks to @Everade for reporting this issue.

### CODE REVIEW

**Comment by llchrisll** [comment]
Well I did it already locally, also you forgot to update the https://github.com/llchrisll/ROenglishRE/blob/master/Addons/jRO%20Enchants%20Display/data/cardpostfixnametable.txt
as well :P

**Comment by Daraen1** [comment]
No worries, I'll do it a bit later in the evening

File: Renewal/data/cardpostfixnametable.txt [hunk 1]
```diff
@@ -1,576 +1,496 @@
...
+300112#
+300113#
+300115#
+300116#
+300117#
+300118#
+300119#
+300125#
+300128#
+300130#
```
Comment (Everade) [inline Renewal/data/cardpostfixnametable.txt:321]: You missed these
```
300128#
300130#
```

**Review by Everade — CHANGES_REQUESTED** [review]

File: Renewal/data/cardpostfixnametable.txt (hunk 1 above)
Comment (Daraen1) [reply Renewal/data/cardpostfixnametable.txt:321]: Sorry and Thanks :)

Comment (Daraen1) [reply Renewal/data/cardpostfixnametable.txt]: update :)

**Pull request merged**